examples/basic/App.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { SafeAreaView } from 'react-native';
+import { LoginForm } from './components/LoginForm';
+import { Home } from './components/Home';
+
+const App = () => {
+  const [user, setUser] = React.useState<string | null>(null);
+
+  if (user == null) {
+    return (
+      <SafeAreaView>
+        <LoginForm onLoginSuccess={setUser} />
+      </SafeAreaView>
+    );
+  }
+
+  return (
+    <SafeAreaView>
+      <Home user={user} />
+    </SafeAreaView>
+  );
+};
```

Comment:
less code is less code
```suggestion
  return (
    <SafeAreaView>
      {user == null ? <LoginForm onLoginSuccess={setUser} /> : <Home user={user} />}
    </SafeAreaView>
  );
};
```